otterdog/eclipse-kuksa.jsonnet

```diff
@@ -54,6 +54,20 @@ orgs.newOrg('eclipse-kuksa') {
         kuksa_default_branch_protection_rule('main')
       ],
     },
+    orgs.newRepo('kuksa-android-companion') {
```

Comment:
Do you not already have settings at https://github.com/eclipse-kuksa/.eclipsefdn/blob/main/otterdog/eclipse-kuksa.jsonnet#L142, i.e. at the end of this file?

Reply:
Never imagined this was already added. :O
Changed it.